examples/standalone_flow_matching.ipynb

```diff
@@ -27,14 +27,19 @@
     "            nn.Linear(h, h), nn.ELU(),\n",
     "            nn.Linear(h, h), nn.ELU(),\n",
     "            nn.Linear(h, dim))\n",
-    "    \n",
+    "\n",
+    "    # Batch-aware forward call\n",
     "    def forward(self, t: Tensor, x_t: Tensor) -> Tensor:\n",
     "        return self.net(torch.cat((t, x_t), -1))\n",
-    "    \n",
-    "    def step(self, x_t: Tensor, t_start: Tensor, t_end: Tensor) -> Tensor:\n",
+    "\n",
+    "    # ODE-solve step for sampling\n",
+    "    def step(self, x_t: Tensor, t_start: Tensor, t_end: Tensor) -> Tensor:      \n",
     "        t_start = t_start.view(1, 1).expand(x_t.shape[0], 1)\n",
-    "        \n",
-    "        return x_t + (t_end - t_start) * self(t=t_start + (t_end - t_start) / 2, x_t= x_t + self(x_t=x_t, t=t_start) * (t_end - t_start) / 2)"
+    "\n",
+    "        # Midpoint method with Delta t = t_end - t_start\n",
+    "        mid_t = t_start + (t_end - t_start) / 2\n",
+    "        mid_x = x_t + self(x_t=x_t, t= t_start) * (t_end - t_start) / 2\n",
+    "        return x_t + self(x_t=mid_x, t=mid_t) * (t_end - t_start)\n"
```

Comment:
Let's make sure that we are only adding comments. This code must remain identical to Code 1 in the paper.